client/landing/stepper/stores.ts

```diff
@@ -24,3 +25,5 @@ export const USER_STORE = User.register( {
 } );
 
 export const AUTOMATED_ELIGIBILITY_STORE = AutomatedTransferEligibility.register();
+
+export const PLANS_STORE = Plans.register();
```

Comment:
Good example of how importing directly from the actual store is useful! This change probably wouldn't have been needed if the code was already using the new redux pattern :)